EDAM terms for Circos, Jbrowse

hexylena opened the pull request:
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

xref https://github.com/galaxyproject/galaxy/pull/10592

bgruening reviewed (approved):
Can you maybe also add the bio.tools identifier? https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-xrefs


Thanks @hexylena!

hexylena commented:
@bgruening sure, added to jbrowse.

I've also added the "real" circos ID to the circos tool which only had our 'galactic circos' identifier. Is that correct to do? Or should we keep it with just the 'galactic circos' id?

bgruening commented on tools/circos/circos.xml:
```diff
@@ -3,6 +3,7 @@
     <description>visualizes data in a circular layout</description>
     <xrefs>
         <xref type="bio.tools">galactic_circos</xref>
+        <xref type="bio.tools">circos</xref>
```
ha, good question. Not sure ... but better to have more annotation than less :)
But we should consult with @hmenager, he has integrated this feature if I remember corectly. Maybe move the main bio.tools ID at the top?

hexylena replied:
The underlying tool is Circos, but our wrapper is a significant addition on top of it, hence having it's own ID, so, yea. Strange :)

bgruening replied:
Changed the ordering, hope that is ok. Ready to merge.

hexylena replied:
It is not time sensitive, happy to wait for @hmenager's comments :)

hmenager commented:
Thanks a lot for pushing this @hexylena !
Here are a few recommendations, up for discussion:
- `EDAM topics` I would have the same for all circos wrappers, since they designate the general subfield that the tool is concerned with. I would annotate circos with `Comparative genomics (topic_0797)`and `Data visualisation (topic_0092)` just like in bio.tools, unless there is a specificity in Galaxy circos which would have it diverge significantly from this. For jbrowse, the topic in bio.tools is `Genomics` and I would add `Data visualisation (topic_0092)`. 
- As a general rule, try to avoid topic_0003 (which is the Topic terms root) for tool annotation.
- `EDAM operations` look fine to me, nothing to say :smiley: 
- `xrefs` are fine, but I would only keep galactic_circos (and not circos) since from what you say @hexylena it is an overlay on circos.

Please tell me if there is anything else I can do to help out.

hmenager commented:
One thing that comes to my mind is that some of these remarks I made could be useful as documentation. I can help out with that as well.

hexylena commented:
Hey @hmenager, that's incredibly useful advice! Thank you !

> remarks I made could be useful as documentation

Yes, definitely!

hmenager commented:
> Yes, definitely!

@hexylena Would a PR for [this documentation page](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#) the most appropriate location?

hexylena commented:
Yeah that's definitely a good place for it!

man. 2. nov. 2020, 17:36 skrev Hervé Ménager <notifications@github.com>:

> Yes, definitely!
>
> @hexylena <https://github.com/hexylena> Would a PR for this documentation
> page
> <https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#>
> the most appropriate location?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/galaxyproject/tools-iuc/pull/3269#issuecomment-720583624>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADP7O3LWLPB3UAZFI3J6TDSN3NYVANCNFSM4TEYYQCQ>
> .
>

bernt-matthias commented:
As a follow up: I just found this old one: https://github.com/galaxyproject/tools-iuc/pull/1897 .. could be interesting to get EDAM for more tools

bgruening commented:
@hexylena is that one ready?

hexylena commented:
Yep

bgruening commented:
Danki!
